### PR TITLE
Unite test case for GetOrCreateNodeRes

### PR DESCRIPTION
**Pull request by HemantGaikwad7**
# Description
Summarize the change.

Fixes #<issue-number> (link all the GitHub issues this addresses)

# Testing
Describe how you tested this code. How can the reviewers reproduce your tests?

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [x] I have self-reviewed this PR.
- [ ] I have removed all print-debugging and commented-out code that should not be merged.
- [ ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: pkg/segment/reader/record/rrcreader_test.go [hunk 1]
```diff
@@ -245,3 +245,43 @@ func Test_GetJsonFromAllRrc_withAggs_ExcludeCols(t *testing.T) {
 
 	os.RemoveAll(dir)
 }
+
+//Variables are defined below
+type NodeResult struct{}
```
Comment (AndrewHess) [inline pkg/segment/reader/record/rrcreader_test.go:250]: Why do you need to make a new type for this unit test? Can't you just use the existing `NodeResult` type from segstructs.go?

**Review by AndrewHess — COMMENTED** [review]
There's several lint issues. You should make sure `make lint` and `make test` return no errors

**Comment by AndrewHess** [comment]
@HemantGaikwad7 do you plan to continue working on this issue? If not, let us know and someone else can take up this task

**Comment by HemantGaikwad7** [comment]
Hi Andrew,

Hope you are doing well.
Please assign it to someone else.

Thanks,
Hemant

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request closed without merging**